web/wp-content/themes/dctx-v2/template-parts/scaffolding/scaffolding-icons.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-alhers.php, but found phpcs-scan-aLhERS.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).